sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/TracerProviderConfiguration.java

```diff
@@ -11,15 +11,24 @@
 import io.opentelemetry.sdk.trace.SdkTracerProviderBuilder;
 import io.opentelemetry.sdk.trace.SpanLimits;
 import io.opentelemetry.sdk.trace.SpanLimitsBuilder;
+import io.opentelemetry.sdk.trace.SpanProcessor;
 import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
 import io.opentelemetry.sdk.trace.export.BatchSpanProcessorBuilder;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
 import io.opentelemetry.sdk.trace.export.SpanExporter;
 import io.opentelemetry.sdk.trace.samplers.Sampler;
 import java.time.Duration;
+import java.util.Collections;
+import java.util.HashSet;
 import java.util.ServiceLoader;
+import java.util.Set;
 
 final class TracerProviderConfiguration {
 
+  /** A set of the exporter names that should use the SimpleSpanProcessor. */
+  private static final Set<String> SPAN_EXPORTER_USING_SIMPLE_SPAN_PROCESSOR =
+      new HashSet<>(Collections.singletonList("logging"));
```

Comment:
this can just all be replaced with `Collections.singleton("logging")`, I think

Reply:
But, I think we can probably just dispense with the set, and just look for the one string directly for now. I don't know if we will ever have more than one hard-coded simple span processor exporter.

Reply:
Ahh gotcha - I wasn't sure what the future plans were. I'll just make it a string